frontend/src/js/concept-trees/ConceptTreeNode.tsx

```diff
@@ -121,6 +88,10 @@ const ConceptTreeNode: FC<PropsT> = ({
             matchingEntities: data.matchingEntities,
             dateRange: data.dateRange,
 
+            excludeTimestamps:
+              root.excludeFromTimeAggregation ||
```

Comment:
to also "pass down" a date exclusion setting from the root to the concepts cc @awildturtok